pages/[[...slug]].js

```diff
@@ -31,26 +41,48 @@ export const getStaticPaths = async () => {
 };
 
 export const getStaticProps = async (context) => {
-    const {
-        params: { slug }
-    } = context;
     try {
-        let url = slug ? `/${slug.join('/')}` : '/index';
+        const {
+            params: { slug }
+        } = context;
+
+        // Determine our head and tail
```

Comment:
Head and tail of what?

Reply:
Head and tail of the slug. I went with a generic name because `head` can be anything instead of a language and `tail` could've been `rest` but head|tail is a good practice: https://coderwall.com/p/jmluka/accessing-head-tail-of-array-es2015